src/Scream.php

```diff
@@ -94,7 +96,7 @@ public function __set($name, $value)
 	 * @param string $name property name
 	 * @throws \Kdyby\StrictObjects\MemberAccessException
 	 */
-	public function __isset($name)
+	public function __isset(string $name): void
```

Comment:
See https://github.com/Kdyby/StrictObjects/pull/4#issuecomment-307527204. PHP defines to return type so this is compatible with PHP and any imlementing code may pick any whatever it wants. :/